xml/cl.xml

```diff
@@ -7313,6 +7321,15 @@ server's OpenCL/api-docs repository.
                 <enum name="CL_STRUCTURE_TYPE_MUTABLE_BASE_CONFIG_KHR"/>
                 <enum name="CL_STRUCTURE_TYPE_MUTABLE_DISPATCH_CONFIG_KHR"/>
             </require>
+            <require comment="cl_command_buffer_properties_khr">
+                <enum name="CL_MUTABLE_DISPATCH_ASSERTS_KHR"/>
+            </require>
+            <require comment="cl_ndrange_kernel_command_properties_khr">
+                <enum name="CL_MUTABLE_DISPATCH_ASSERTS_KHR"/>
```

Comment:
would CL_COMMAND_MUTABLE_DISPATCH_ASSERTS be a better name?

Reply:
But all other names begin with "CL_MUTABLE_DISPATCH_".